### DISCUSSION TITLE
Walmart monitor does not work after Walmart website update

### DISCUSSION THREAD

**Discussion by xmpierce1** [Prince25/StockAlertBot#119] [Q&A] [answered]
Walmart updated its website with Buy Now and Add to Cart so the URLs are not detecting stock anymore due to the change. 

Sample URL below:

https://www.walmart.com/ip/AirTag-Leather-Key-Ring-Wisteria/798472488

**Comment by Prince25** [comment, marked as answer]
Discusssed [here](https://github.com/Prince25/StockAlertBot/discussions/119).
